### ISSUE TITLE
Accessibility - select cannot be focussed with Voice Control on mac when in item numbers mode

### ISSUE THREAD

**Issue by k-sav** [JedWatson/react-select#4934]
![react-select-issue](https://user-images.githubusercontent.com/5423575/143790447-3dc553dd-21b9-4374-a7fa-2cd4900b6c34.png)

In the screenshot above, while using Voice Control on mac when in **Item numbers** mode you can see that every interactive element has a number assigned to it which can be _clicked_ by the user when they say that number.

There is no number next to the select because the html input is hidden, and there is no other way that Assistive tech can determine that the replaced `.select__control` is interactive.

As such, a user using Voice Control on mac would need to switch to **Numbered grid** mode to be able to open the select.
